superset/assets/javascripts/explore/stores/visTypes.js

```diff
@@ -622,6 +624,37 @@ const visTypes = {
       },
     },
   },
+  chord: {
```

Comment:
This file contains configuration of the new `chord` viz type. It defines the panels of controls that should render on the left panel of the explore view. 

`controlSetRows` references controls defined in `controls.jsx`, and can have one or two items and that defines how many items will render on that row.

Reply:
> This file contains configuration of the new chord viz type. It defines the panels of controls that should render on the left panel of the explore view.
> controlSetRows references controls defined in controls.jsx, and can have one or two items and that defines how many items will render on that row.

Could I please ask some questiones? I want to show a metric and some filter in the superset dashboard, after reading some articles, i am confused because the configuration needed to be changed does not exist in the files. For example,
superset/assets/javascripts/explore/stores/visTypes.js, this file does not exist in the current code file. So, how can i add a metric in the superset now and which files should i modify in order to achieve this?